for_challenges.py

```diff
@@ -25,8 +25,12 @@
     'Маша': False,
 }
 names = ['Оля', 'Петя', 'Вася', 'Маша']
-# ???
-
+for name in names:
+    if is_male[name]:
```

Comment:
Научу плохому. Можно так сделать:
```suggestion
    gender = "жм"[is_male[name]]
```
Работает потому что False это 0, и мы берем элемент строки с нулевым индексом. 
Можно даже:
```suggestion
    gender = ("woman", "man")[is_male[name]]
```

Reply:
Офигенно!!)